CHANGELOG.unreleased.md

```diff
@@ -31,6 +33,7 @@
 
 <!--packages-start-->
 
+- @vates/nbd-client patch
```

Comment:
If it's a patch, and it fix race condition, maybe the PR should be fix(nbd), not feat(nbd) ?

Reply:
I agree with you, even if it's not a bug in our lib itself.